### PR TITLE
EVG-7741 add route to update roles

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: rest/route/user.go [hunk 1]
```diff
@@ -364,3 +364,90 @@ func (h *userPermissionsGetHandler) Run(ctx context.Context) gimlet.Responder {
...
+	}
+}
+
+func (h *userRolesPostHandler) Parse(ctx context.Context, r *http.Request) error {
+	vars := gimlet.GetVars(r)
+	h.userID = vars["user_id"]
+	if h.userID == "" {
+		return errors.New("no user found")
+	}
+	roles := []string{}
```
Comment (tychoish) [inline rest/route/user.go:395]: I think you can ommit this. and just &h.roles in the next line.

File: rest/route/user.go [hunk 2]
```diff
@@ -364,3 +364,90 @@ func (h *userPermissionsGetHandler) Run(ctx context.Context) gimlet.Responder {
...
+		sc: h.sc,
+		rm: h.rm,
+	}
+}
+
+func (h *userRolesPostHandler) Parse(ctx context.Context, r *http.Request) error {
+	vars := gimlet.GetVars(r)
+	h.userID = vars["user_id"]
+	if h.userID == "" {
+		return errors.New("no user found")
```
Comment (tychoish) [inline rest/route/user.go:393]: I think it's impossible to produce a request that would get routed to this request, given how the muxer works.

Comment (john-m-liu) [reply rest/route/user.go]: removed

File: rest/route/user.go (hunk 1 above)
Comment (john-m-liu) [reply rest/route/user.go]: done

**Pull request merged**